Resources/views/Collector/db.html.twig

```diff
@@ -490,15 +498,15 @@
             });
 
             for (i = 0; i < items.length; ++i) {
-                Sfjs.removeClass(items[i], i % 2 ? 'even' : 'odd');
-                Sfjs.addClass(items[i], i % 2 ? 'odd' : 'even');
+                items[i].classList.remove(i % 2 ? 'even' : 'odd');
+                items[i].classList.add(i % 2 ? 'odd' : 'even');
```

Comment:
I think we can just remove them, `even` and `odd` classes seem to be no longer used.